beacon-chain/rpc/prysm/v1alpha1/validator/proposer_phase0.go

```diff
@@ -82,6 +82,10 @@ func (vs *Server) buildPhase0BlockData(ctx context.Context, req *ethpb.BlockRequ
 		return nil, fmt.Errorf("syncing to latest head, not ready to respond")
 	}
 
+	if err := vs.HeadUpdater.ProcessAttestationsAndUpdateHead(ctx); err != nil {
+		log.WithError(err).Error("Could not process attestations and update head")
```

Comment:
Opt it to log an error instead of return error. Error from this call should not prevent block proposal